client/components/Agendas/AgendaItem.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import moment from 'moment';
+import {TOOLTIPS} from '../../constants';
+import {List} from '../UI';
+
+export const AgendaItem = ({agenda, deleteAgenda, editAgenda, privileges, active}) => (
+    <List.Item shadow={1}>
+        <List.Border state={active ? 'active' : 'idle'} />
+        <List.Column grow={true} border={false}>
+            <List.Row>
+                <span className="sd-overflow-ellipsis sd-list-item--element-grow">{agenda.name}</span>
+                <time>updated {moment(agenda._updated).fromNow()}</time>
+            </List.Row>
+        </List.Column>
+        {privileges.planning_agenda_management === 1 &&
+            <div className="sd-list-item__action-menu sd-list-item__action-menu--direction-row">
```

Comment:
There is a `List.ActionMenu` component that you could use here, and adding a prop to control the `sd-list-item__action-menu--direction-row` CSS class